TF_Service_dotNet/TouchFree/VirtualScreen.cs

```diff
@@ -64,7 +64,8 @@ public float DistanceFromScreenPlane(Vector3 worldPosition)
 
             RaycastAgainstPlane(PhysicalScreenPlane, r, out float distanceFromPlane);
 
-            return distanceFromPlane;
+            // Take the absolute value here to normalise against negative values
+            return Math.Abs(distanceFromPlane);
```

Comment:
I think we may still want to allow the return of negative values here? it would signify the user is past the virtual screen plane which:

- could be useful to know in certain circumstances
- would still result in the 'nearest bone being the one furthest from the user (if stood in front of the virtual screen facing it)

Reply:
I have removed this absolute and resolved an issue with positioning in the code